psgdpr.php

```diff
@@ -625,7 +625,7 @@ public function hookDisplayCustomerAccount()
             'ps_version' => $this->ps_version,
         ]);
 
-        return $this->display(dirname(__FILE__), '/views/templates/front/customerAccount.tpl');
+        return $this->fetch('module:' . $this->name . '/views/templates/front/customerAccount.tpl');
```

Comment:
It can stay like this with the display and dirname. It just needs to skip the first slash for the views folder.

```suggestion
        return $this->display(dirname(__FILE__), 'views/templates/front/customerAccount.tpl');
```

Reply:
Hi @JoeyJanson , noted :) Also PR to this another module facing the same problem with the slash before the views folder

https://github.com/PrestaShopCorp/ps_checkout/pull/829